.circleci/config.yml

```diff
@@ -179,7 +182,7 @@ jobs:
       - log_stats:
           file: prepare-stats
       - <<: *restore_dep
-      - run: yarn --frozen-lockfile
+      - run: yarn --frozen-lockfile --cache-folder ~.cache/yarn
```

Comment:
Not sure I understand the reason for specifying the `cache-folder`

Reply:
There's an explanation here: https://circleci.com/docs/2.0/caching/#yarn-node. 

I thought it was worth putting in, in case it made caching of dependencies work better. It seems to me that caching is not working great at the moment, since I don't think I have ever seen `success Already up-to-date.` on circle, which I would have expected?

Reply:
Is the `~.cache/yarn` directory persisted between workflow runs?

Reply:
No! good catch

Reply:
https://github.com/statechannels/statechannels/pull/2558 💥